main.tf

```diff
@@ -104,19 +104,23 @@ resource "google_compute_shared_vpc_service_project" "shared_vpc_attachment" {
   Default compute service account retrieval
  *****************************************/
 data "google_compute_default_service_account" "default" {
-  project    = "${local.project_id}"
-  depends_on = ["google_project_service.project_services"]
```

Comment:
@ryanckoch Why is this dependency removed? The default compute service account won't appear until the compute API is activated.

Reply:
Because of an issue where it causes the dependent trigger to run on every apply:
https://github.com/hashicorp/terraform/issues/11806